README.md

```diff
@@ -1,5 +1,10 @@
 # Observability Libs
 
+[![CharmHub Badge](https://charmhub.io/observability-libs/badge.svg)](https://charmhub.io/observability-libs)
```

Comment:
This is a charm badge, and this charm is a tester/dummy one.
We probably can skip the charm badge for utils/relation repos.

Reply:
True, removing that